assets/js/util.js

```diff
@@ -119,6 +119,25 @@ $(document).ready(function () {
     $('.modal').on('hide.bs.modal', function () {
         $('a[data-target=#' + $(this).attr('id') + ']').parents('li').removeClass('active');
     });
+
+    $(window).scroll(function () {
+        if($(window).scrollTop() > BACK_TO_TOP_ACTIVATION_HEIGHT) {
+            $('#backToTop').removeClass('hidden-xs hidden-sm hidden-md hidden-lg');
+            $('#backToTop').addClass('hidden-xs hidden-sm visible-md visible-lg');
```

Comment:
Um, if we do not explicitly mention `hidden-xs hidden-sm`, would it appear in the small screen devices too? Because, otherwise, there is no place where I explicitly mention that this feature should be hidden on mobile! Maybe I am wrong!